gitless/tests/utils.py

```diff
@@ -82,6 +82,22 @@ def onerror(func, path, unused_exc_info):  # error handler for rmtree
   logging.debug('Removed dir {0}'.format(path))
 
 
+def symlink(src, dst, target_is_directory=False, dir_fd=None):
+  try:
+    os.symlink(src, dst, target_is_directory, dir_fd=dir_fd)
```

Comment:
In python 2 the `symlink` method has a [different signature](https://docs.python.org/2/library/os.html#os.symlink). It doesn't look like you are using `target_is_directory` and `dir_fd`. Perhaps we can remove those parameters? Then tests should pass on python 2 and 3